instructions/section-2/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
I think this file was supposed to be created in _just_ the `intro-to-programming` and not in the `instructions/section-2` directory